apps/laboratory/src/components/Ethers/EthersSendCallsTest.tsx

```diff
@@ -133,23 +177,62 @@ export function EthersSendCallsTest() {
   }
 
   return currentChainsInfo ? (
-    <Stack direction={['column', 'column']}>
-      <Button
-        data-testid="send-calls-button"
-        onClick={onSendCalls}
-        isDisabled={loading}
-        maxWidth={'50%'}
-      >
-        Send Batch Calls to Vitalik
-      </Button>
-      <Spacer />
+    <>
+      <Box>
+        <Stack direction={['column', 'column', 'row']}>
+          <Stack direction={['column']}>
+            (
+            {transactionsToBatch.length ? (
+              transactionsToBatch.map((tx, index) => (
+                <>
+                  <Card>
+                    <CardBody>
+                      <Stat>
+                        <StatLabel>
+                          ({index + 1}) Sending
+                          <DeleteIcon
+                            style={{ float: 'right', cursor: 'pointer' }}
+                            onClick={() => onRemoveTransaction(index)}
```

Comment:
Better avoid removing items from the index and better the tx hash.

Reply:
there is no transaction hash at this point, I can hash the payload and use that but given its just for example transactions no need to overcomplicate it